04 Strategy Library/999 Seasonality and Session Effect in Index Futures/02 Strategy Theory.html

```diff
@@ -0,0 +1,3 @@
+<p>
+	The logic of this strategy is to buy at the open of the session and close that trade at the conclusion of the session. In this tutorial, we will show the implementation of the algorithm on QuantConnect, and study the performance of Combined Strategy, and the performance of Night Session Strategy on Bearish Season and Bullish Season. 
```

Comment:
The underlying logic of this strategy is to buy at the open of a session and close our position at the conclusion of the session. Given that there are multiple profitable sessions, it is natural to come up with a combined strategy which aggregates the top three profitable bullish and the top two profitable bearish strategies to see if we can maximize the returns. In this tutorial, we will show the implementation of the algorithm on QuantConnect, and study the performance of Combined Strategy, and the performance of Night Session Strategy on Bearish Season and Bullish Season.